docs/protocol-p2p-network/initial-block-download.md

```diff
@@ -19,3 +19,125 @@ best block chain has a block header time more than 24 hours in the past.
 the local block chain is more than about 24 hours in the past).
+
+## Blocks-First
+
+dcrd uses a simple initial block download (IBD) method called *blocks-first*.
+The goal is to download the blocks from the best block chain in sequence.
+
+![Overview Of Blocks-First Method](/img/protocol-p2p-network/en-blocks-first-flowchart.svg)
+
+The first time a node is started, it only has a single block in its
+local best block chain---the hardcoded genesis block (block 0).  This
+node chooses a remote peer, called the sync node, and sends it the
+`getblocks` message illustrated below.
+
+![First GetBlocks Message Sent During IBD](/img/protocol-p2p-network/en-ibd-getblocks.svg)
+
+In the header hashes field of the `getblocks` message, this new node
+sends the header hash of the only block it has, the genesis block
+(6fe2...0000 in internal byte order).  It also sets the stop hash field
+to all zeroes to request a maximum-size response.
+
+Upon receipt of the `getblocks` message, the sync node takes the first
+(and only) header hash and searches its local best block chain for a
+block with that header hash. It finds that block 0 matches, so it
+replies with 500 block inventories (the maximum response to a
+`getblocks` message) starting from block 1. It sends these inventories
+in the `inv` message illustrated below.
+
+![First Inv Message Sent During IBD](/img/protocol-p2p-network/en-ibd-inv.svg)
+
+Inventories are unique identifiers for information on the network. Each
+inventory contains a type field and the unique identifier for an
+instance of the object. For blocks, the unique identifier is a hash of
+the block's header.
+
+The block inventories appear in the `inv` message in the same order they
+appear in the block chain, so this first `inv` message contains
+inventories for blocks 1 through 501. (For example, the hash of block 1
+is 4860...0000 as seen in the illustration above.)
+
+The IBD node uses the received inventories to request 128 blocks from
+the sync node in the `getdata` message illustrated below.
+
+![First GetData Message Sent During IBD](/img/protocol-p2p-network/en-ibd-getdata.svg)
+
+It's important to blocks-first nodes that the blocks be requested and
+sent in order because each block header references the header hash of
+the preceding block. That means the IBD node can't fully validate a
+block until its parent block has been received. Blocks that can't be
+validated because their parents haven't been received are called [orphan
+blocks](orphan-blocks.md).
+
+Upon receipt of the `getdata` message, the sync node replies with each
+of the blocks requested. Each block is put into serialized block format
+and sent in a separate `block` message. The first `block` message sent
+(for block 1) is illustrated below.
+
+![First Block Message Sent During IBD](/img/protocol-p2p-network/en-ibd-block.svg)
+
+The IBD node downloads each block, validates it, and then requests the
```

Comment:
What is IBD node? So far, we've only referred to the IBD 'method'. This is the first time IBD node is used and needs to be defined.